frontend/src/components/dialogs/InfoDialog.vue

```diff
@@ -0,0 +1,120 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+ <template>
+  <g-dialog
+    ref="gDialog"
+    confirm-button-text="Ok"
+    cancel-button-text=""
+    maxWidth="600"
+    @dialog-closed="onDialogClosed()"
+    >
+    <template v-slot:caption>About Gardener Dashboard</template>
+    <template v-slot:message>
+      <div class="d-flex flex-row align-center mt-3">
+        <img src="../../assets/logo.svg" class="logo mr-3">
+        <div>
+          <h2 class="mb-1">Gardener Dashboard</h2>
+        </div>
+      </div>
+      <v-divider class="my-3"></v-divider>
+      <div class="grey--text text--darken-1">
+        <div class="font-weight-bold">Version Information</div>
+        <div v-if="!!dashboardVersion">Dashboard version {{dashboardVersion}}</div>
+        <div v-if="!!gardenerVersion">API version {{gardenerVersion}}</div>
```

Comment:
```suggestion
        <div v-if="!!dashboardVersion">Dashboard {{dashboardVersion}}</div>
        <div v-if="!!gardenerVersion">API {{gardenerVersion}}</div>
```
omit "version"?  Same as it is for the extensions